data/pilots/rebel-alliance/t-65-x-wing.json

```diff
@@ -407,6 +407,43 @@
       "extended": true,
       "keywords": ["X-wing"],
       "epic": true
+    },
+    {
+      "name": "Corran Horn",
+      "caption": "Rogue Nine",
+      "initiative": 5,
+      "limited": 1,
+      "cost": 0,
+      "loadout": 0,
+      "xws": "corranhorn-t65xwing",
+      "ability": "After you declare the defender of an attack, if you have a lock on the defender, a friendly ship may transfer their lock token from you to the defender.",
+      "slots": [
+      ],
+      "image": "https://infinitearenas.com/xw2/images/pilots/corranhorn-t65xwing.png",
+      "artwork": "",
+      "standard": true,
+      "extended": true,
+      "keywords": ["X-wing"],
+      "epic": true
+    },
+    {
+      "name": "Wes Janson",
+      "caption": "Wisecracking Wingman",
+      "initiative": 5,
+      "limited": 1,
+      "cost": 0,
+      "loadout": 0,
+      "xws": "wesjanson",
+      "ability": "After you perform an attack, you may spend 1 [Charge] to assign the defender 1 jam token. After you defend, you may spend 1 [Charge] to assign the attacker 1 jam token.",
```

Comment:
@danrs some of these abilities have either newlines in them, or bold characters (like obi/ani eta2). I didn't see any other cards using special formatting for that, is that something I missed or is it just not represented in xws?

Reply:
There have been other cards like that (see the Magnaguard protector in the rogue for a bold example, I can't think of a newline example). We simply do not represent newlines or formatted text in the XWS ability string.